content/numpy/concepts/math-methods/terms/nanprod/nanprod.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: '.nanprod()'
+Description: 'Calculates the product of array elements over a given axis, treating NaN (Not a Number) values as 1.'
```

Comment:
```suggestion
Description: 'Calculates the product of array elements.'
```

Reply:
reframed as this gives a basic description and later in-depth explanation is given